src/org/labkey/test/pages/core/admin/BaseSettingsPage.java

```diff
@@ -0,0 +1,303 @@
+package org.labkey.test.pages.core.admin;
+
+import org.labkey.test.Locator;
+import org.labkey.test.components.html.RadioButton;
+import org.labkey.test.pages.LabKeyPage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class BaseSettingsPage extends LabKeyPage<LabKeyPage<?>.ElementCache>
```

Comment:
If you change how this is declared, you won't have to override `elementCache` below.
```suggestion
public class BaseSettingsPage extends LabKeyPage<BaseSettingsPage.ElementCache>
```

Reply:
Changed.